src/PlotKit.jl

```diff
@@ -51,6 +56,10 @@ export hbox, vbox, hvbox, stack, offset
 # animator.jl
 export Anim, frame, see, save_frames
     
+#------------------------------------------------------------------------------
+# rexport from Colors
+
+export Color, Color3, Color4
```

Comment:
I'm surprised that you have to build your own Color library. I can see that the Cairo C API doesn't directly support colors, although the Ruby version does -- but then Ruby does provide everything :). There is a JuliaGraphics Colors package though: https://github.com/JuliaGraphics/Colors.jl. Any reason not to use it?

Reply:
The colors package uses 8 bit color, instead of floats, which is good for images but not for settings where one computes with color. (copied from above.)